### PR TITLE
Revert suppress FIB pending feature

### PR DESCRIPTION
**Pull request by stepanblyschak**
#### Why I did it

Revert BGP suppress FIB pending due to unresolved FRR issues in current version

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

Revert it

#### How to verify it

Build and run

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: src/sonic-yang-models/yang-models/sonic-device_metadata.yang [hunk 1]
```diff
@@ -212,18 +212,6 @@ module sonic-device_metadata {
                                  By default SONiC advertises /128 subnet prefix in Loopback0 as /64 subnet route";
                 }
 
-                leaf suppress-fib-pending {
```
Comment (lguohan) [inline src/sonic-yang-models/yang-models/sonic-device_metadata.yang:215]: i suggest to keep these yang model instead of simple reverting.

Comment (stepanblyschak) [reply src/sonic-yang-models/yang-models/sonic-device_metadata.yang:215]: @lguohan I don't want to keep CLI and YANG model for a feature that does not work in a release

**Comment by liat-grozovik** [comment]
@lguohan how do you want to followup on that?

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Comment by StormLiangMS** [comment]
hi @stepanblyschak  I think we also need this for 202405, could you help to run a test before we do the cherry pick for 202405? @bingwang-ms  for vis.

**Comment by dgsudharsan** [comment]
@bingwang-ms Please cherry-pick this PR for 202405

**Comment by dgsudharsan** [comment]
@bingwang-ms Can you please check why PR for 202405 is not created?

**Comment by liushilongbuaa** [comment]
> @bingwang-ms Can you please check why PR for 202405 is not created?

I checked the logs. But I don't find the reason.

**Pull request merged**